### PR TITLE
x11-misc/: EAPI bumps

### PR DESCRIPTION
**Pull request by Moiman**

### CODE REVIEW

File: x11-misc/xkbd/xkbd-0.8.16-r1.ebuild [hunk 1]
```diff
@@ -42,13 +38,11 @@ src_configure() {
 
 src_compile() {
 	default
-
 	use doc && docbook2html README
```
Comment (SoapGentoo) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: turn this into
```
if use doc; then
	docbook2html README || die
	HTML_DOCS=( *.html )
fi
```
then nix the whole `src_install` phase

File: x11-misc/xkbd/xkbd-0.8.16-r1.ebuild [hunk 2]
```diff
@@ -27,13 +26,10 @@ DEPEND="${RDEPEND}
...
-	# 2008-03-23 gi1242: Fix handling of -geometry argument
-	epatch "${FILESDIR}"/${P}-fix-geometry.patch
-	# 2008-03-23 gi1242: Increase default repeat delay
-	epatch "${FILESDIR}"/${PN}-0.8.15-increase-delay.patch
-	epatch "${FILESDIR}"/${PN}-0.8.15-fix-keysyms-search.patch
-}
+PATCHES=( "${FILESDIR}"/${P}-fix-geometry.patch
+	"${FILESDIR}"/${P}-desktop.patch
+	"${FILESDIR}"/${PN}-0.8.15-increase-delay.patch
+	"${FILESDIR}"/${PN}-0.8.15-fix-keysyms-search.patch )
```
Comment (SoapGentoo) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: In general, can you prefer:
```
PATCHES=(
	"${FILESDIR}"/${P}-fix-geometry.patch
	"${FILESDIR}"/${P}-desktop.patch
	"${FILESDIR}"/${PN}-0.8.15-increase-delay.patch
	"${FILESDIR}"/${PN}-0.8.15-fix-keysyms-search.patch
)
```
if there is **more than one** patch, as having them all at the same column makes things slightly easier to read

File: x11-misc/accessx/accessx-0951-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+src_prepare() {
+	default
+	sed -i \
+		-e 's:$(CC) $(OPTS) ax.C:$(CC) $(LDFLAGS) $(OPTS) ax.C:' \
+		Makefile || die
+}
+
+src_compile() {
+	emake CC="$(tc-getCXX)" OPTS="${CXXFLAGS}" \
+		XLIBDIR="-L/usr/$(get_libdir)" LLIBS="-lXext -lX11"
```
Comment (SoapGentoo) [inline x11-misc/accessx/accessx-0951-r2.ebuild]: can you make this slightly more portable by using pkg-config? i.e. try if the following works
```
emake CC="$(tc-getCXX)" OPTS="${CXXFLAGS}" XLIBDIR="-L/usr/$(get_libdir)" \
	LLIBS="$($(tc-getPKG_CONFIG) --libs xext) $($(tc-getPKG_CONFIG) --cflags x11)"
```
This is more robust to changes of libXext and libX11, and while unlikely, will make it less likely to fail, should these libs introduce additional requirements.

File: x11-misc/xkbd/xkbd-0.8.16-r1.ebuild (hunk 1 above)
Comment (Moiman) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: docbook2html command seems to fail with a lot of error messages, but it produces some working html files. What should be done about it. Remove die or nuke whole doc useflag?

Comment (SoapGentoo) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: nah, nuke it then. Hoping on some `.html` files is too pointless then imo

File: x11-misc/accessx/accessx-0951-r2.ebuild (hunk 3 above)
Comment (Moiman) [inline x11-misc/accessx/accessx-0951-r2.ebuild]: --cflags x11, doesn't detect anything. I replace it with --libs x11

Comment (SoapGentoo) [inline x11-misc/accessx/accessx-0951-r2.ebuild]: sorry yes, that was incorrectsorry yes, that was incorrect

File: x11-misc/xkbd/xkbd-0.8.16-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	x11-libs/libXtst
+	x11-libs/libXpm
+	media-libs/freetype
+	dev-libs/expat
+	sys-libs/zlib"
+DEPEND="${RDEPEND}
+	x11-proto/xproto
+	x11-proto/xextproto"
+
+DOCS=( AUTHORS )
```
Comment (SoapGentoo) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: you sure `AUTHORS` is required?

File: x11-misc/devilspie/devilspie-0.23.ebuild [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+	sys-devel/gettext
+	gnome-base/gnome-common" # Required by eautoreconf
+
+src_prepare() {
+	default
+	sed -i -e "s:\(/usr/share/doc/devilspie\):\1-${PVR}:" devilspie.1 || die
+	sed -i -e '/-DG.*_DISABLE_DEPRECATED/d' src/Makefile.am || die
+	mv configure.{in,ac} || die
+	eautoreconf
+	export LIBS="-lX11"
```
Comment (SoapGentoo) [inline x11-misc/devilspie/devilspie-0.23.ebuild]: you could also do `LIBS="$($(tc-getPKG_CONFIG) --libs x11)"` (don't forget `toolchain-funcs`) here

File: x11-misc/xkbd/xkbd-0.8.16-r1.ebuild (hunk 4 above)
Comment (Moiman) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: README file is that broken docbook file. I don't want it to be installed.

Comment (SoapGentoo) [inline x11-misc/xkbd/xkbd-0.8.16-r1.ebuild]: now I got it, A-ok then 👍

**Pull request closed without merging**